lib/ruby_indexer/test/method_test.rb

```diff
@@ -149,6 +149,48 @@ def bar; end
       end
     end
 
+    def test_private_class_method_visibility_tracking_string_symbol_arguments
+      index(<<~RUBY)
+        class Test
+          def self.foo
+          end
+
+          def self.bar
+          end
+
+          private_class_method("foo", :bar)
+        end
+      RUBY
+
+      ["foo", "bar"].each do |keyword|
+        entries = T.must(@index[keyword])
+        assert_equal(1, entries.size)
+        entry = entries.first
+        assert_equal(Entry::Visibility::PRIVATE, entry.visibility)
```

Comment:
```suggestion
        assert_predicate(entry.visibility, :private?)
```
(I updated some other occurrences in https://github.com/Shopify/ruby-lsp/pull/2859)